posit-dev/positron
# Build Positron for macOS on EC2

### Intent

Builds Positron for macOS on EC2.

This change makes the build scripts compatible with the EC2 environment. After it is merged, we will ask IT to add the `self-hosted-production` label to the EC2 runner (and remove it from MacInCloud) so EC2 begins absorbing work.

### Approach

Pretty small delta, in the end; though it took a very long time to set up the EC2 runner, it was 95% preconfiguration. Changes necessary:

- Homebrew is now in a new path, so need to point to it there
- No need for a revive-agent job any more (IT manages the runner agent)
- Jobs are now more independently runnable

### QA Notes

Doesn't affect other builds; sanity check of macOS after this change lands is sufficient.
